### PR TITLE
fix: Fix copybook definition lookup

### PR DESCRIPTION
**Pull request by temanbrcom**

### CODE REVIEW

File: server/src/test/java/org/eclipse/lsp/cobol/core/semantics/outline/RangeUtilsTest.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
 package org.eclipse.lsp.cobol.core.semantics.outline;
```
Comment (grianbrcom) [inline server/src/test/java/org/eclipse/lsp/cobol/core/semantics/outline/RangeUtilsTest.java:16]: You can move the test to `delegates.util` package too.

**Review by grianbrcom — APPROVED** [review]
Nice catch! :mag:

File: server/src/test/java/org/eclipse/lsp/cobol/core/semantics/outline/RangeUtilsTest.java (hunk 1 above)
Comment (temanbrcom) [reply server/src/test/java/org/eclipse/lsp/cobol/core/semantics/outline/RangeUtilsTest.java]: Oops, I missed that.

**Review by grianbrcom — APPROVED** [review]
:maple_leaf:

**Pull request merged**